apps/aecore/lib/aecore/keys/worker.ex

```diff
@@ -40,7 +41,8 @@ defmodule Aecore.Keys.Worker do
 
   """
   @spec sign_tx(binary(), integer(), integer(), integer(), integer()) :: {:ok, %SignedTx{}}
-  def sign_tx(to_acc, value, nonce, fee, lock_time_block) do
+  def sign_tx(to_acc, value, nonce, fee,
+              lock_time_block \\ Chain.latest_block().header.height + 1) do
```

Comment:
why do you want to lock a transaction by default? I would set the default to `0`